examples/blog/config.js

```diff
@@ -0,0 +1,220 @@
+                    month = now.getMonth() + 1,
+                    day = now.getDate();
+                month = month < 10 ? '0' + month : month;
+                day = day < 10 ? '0' + day : day;
+                return {
+                    created_at: [year, month, day].join('-') // ?created_at=... will be appended to the API call
+                };
+            }),
+            nga.field('post_id', 'reference')
+                .label('Post')
+                .targetEntity(post)
+                .targetField(nga.field('title'))
+        ])
+        .listActions(['edit', 'delete']);
+
+    comment.views['CreateView']
+        .fields([
+            nga.field('created_at', 'date')
+                .label('Posted')
+                .defaultValue(new Date()), // preset fields in creation view with defaultValue
+            nga.field('author'),
+            nga.field('body', 'wysiwyg'),
+            nga.field('post_id', 'reference')
+                .label('Post')
+                .map(truncate)
+                .targetEntity(post)
+                .targetField(nga.field('title'))
+        ]);
+
+    comment.views['EditView']
+        .fields(comment.views['CreateView'].fields())
+        .fields([nga.field(null, 'template')
+            .label('')
+            .template('<post-link entry="entry"></post-link>') // template() can take a function or a string
+        ]);
+
+    comment.views['DeleteView']
+        .title('Deletion confirmation'); // customize the deletion confirmation message
+
+    tag.views['MenuView']
+        .order(3)
+        .icon('<span class="glyphicon glyphicon-tags"></span>');
+
+    tag.views['DashboardView']
+        .title('Recent tags')
+        .order(3)
+        .limit(10)
+        .fields([
+            nga.field('id'),
+            nga.field('name'),
+            nga.field('published', 'boolean').label('Is published ?')
+        ]);
+
+    tag.views['ListView']
+        .infinitePagination(false) // by default, the list view uses infinite pagination. Set to false to use regulat pagination
+        .fields([
+            nga.field('id', 'number').label('ID'),
+            nga.field('name'),
+            nga.field('published', 'boolean').cssClasses(function (entry) { // add custom CSS classes to inputs and columns
+                if (entry.values.published) {
+                    return 'bg-success text-center';
+                }
+                return 'bg-warning text-center';
+            }),
+            nga.field('custom', 'template')
+                .label('Upper name')
+                .template(function (entity) {
+                    entity.name.toUpperCase();
+                })
+        ])
+        .listActions(['show']);
+
+    tag.views['ShowView']
+        .fields([
+            nga.field('name'),
+            nga.field('published', 'boolean')
+        ]);
+
+    React.render(
+        React.createElement(ReactAdmin, {configuration: admin}),
+        document.body
+    );
```

Comment:
Is it possible to do only one wrapper `ReactAdminConfiguration` which contains all methods of Factory and a method `configure` (similar to ng-admin) ?